app/templates/inbox/irg-archive.hbs

```diff
@@ -4,10 +4,11 @@
   <table.menu as |menu|>
     <menu.general>
       <AuToolbar class="au-o-box" @size="large" as |Group|>
-        <Group>
+        <Group class="au-c-toolbar__group--vertical">
```

Comment:
I like this! I would've added an extra div for no reason, the way you did it is better :)